perceval/__init__.py

```diff
@@ -17,7 +17,8 @@
 # Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307, USA.
 #
 # Authors:
-#     Santiago Dueñas <sduenas@bitergia.com>
+#       Jesus M. Gonzalez-Barahona <jgb@bitergia.com>
```

Comment:
No idea. Sorry about that. Probably I used some template of mine or something :-( I'll revert that.